Dockerfile

```diff
@@ -60,6 +60,20 @@ RUN GO111MODULE=on go get \
 
 RUN apt-get install -yq clang-format
 
+# Minio mint test suite
+RUN apt-get install -y wget jq curl git dnsmasq python3-dateutil
+RUN git init
```

Comment:
Ensure that this happens in a unique directory, otherwise this might be creating a new git repository in home directory.